### PR TITLE
chore: bump kube-vip to v0.8.7

### PR DESCRIPTION
**Pull request by starbops**
Bumping harvester-cloud-provider's dependency - kube-vip:

- Chart version: 0.6.4
- App version: v0.8.7

Related issue: harvester/harvester#6904

### CODE REVIEW

**Review by bk201 — APPROVED** [review]

**Comment by w13915984028** [comment]
@starbops Could you please also help: 
(1) Merge PR 305 into your PR, to allow disable Kube-vip
(2) Cut a new CPH image v0.2.4 to include the latest commits

thanks.

**Review by w13915984028 — APPROVED** [review]
LGTM, thanks.

**Comment by bk201** [comment]
@w13915984028 my suggestion is to re-open https://github.com/harvester/charts/pull/279 and merge it first then we can merge this one.

**Comment by starbops** [comment]
I've added the patch for the kube-vip switch. After this PR is merged, I'll close #279, #305, and #306. Thank you. cc @w13915984028 @adonskoy

**Review by FrankYang0529 — APPROVED** [review]

File: charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/_helpers.tpl [hunk 1]
```diff
@@ -63,12 +63,18 @@ Create the name of the service account to use
 {{- end }}
 
 {{/*
-Global system default registry
+Convert string to boolean
 */}}
-{{- define "system_default_registry" -}}
-{{- if .Values.global.cattle.systemDefaultRegistry -}}
-{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
```
Comment (brandond) [inline charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/_helpers.tpl:70]: This has broken support for system default registry, and will need to be fixed before this update can go into Rancher or RKE2.

Comment (Vicente-Cheng) [reply charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/_helpers.tpl:70]: Thanks @brandond. 
We are working on https://github.com/harvester/charts/pull/322 for this issue.

cc @FrankYang0529, @bk201

**Pull request merged**